docs/student.md

```diff
@@ -103,7 +103,7 @@ The organizing committee does not recommend that students start research and dev
   The students’ bonus will be paid by credit transfer method. There is no restriction on which bank but the bank account must be under your own name. Also, the organizing committee will collect the following information from the mentor:
 
   - Name
-  - ID number
+  - Citizen ID No.
   - Bank account and branch information
```

Comment:
@YifeiXia please confirm with the finance team whether the information of mobile number is also necessary